### PR TITLE
Cmd update

### PR DESCRIPTION
**Pull request by notzippy**
Refactored cmd functions to provide improvement
1) Refactored new so other new commands can be generated from it (ie new raml or the likes) start for #73
2) Added ability to detect file location even if it is on a symlink gopath. 
3) Added option to package to include no source files #49

### CODE REVIEW

File: revel/package.go [hunk 1]
```diff
@@ -5,65 +5,92 @@
...
+                    !strings.Contains(shortPath,"/")
+
+            }
+        }
+
+    }
+
+
+
+    buildTheApp(args[0],tmpDir,mode)
```
Comment (brendensoares) [inline revel/package.go]: Why did this change to `buildTheApp` instead of `buildApp`?

Comment (notzippy) [inline revel/package.go]: `buildApp(args []string)` works from the command line arguments and makes assumptions based on the order of them. `buildTheApp(appImportPath, destPath, mode string)` is specific in the argument order and requirements

Comment (brendensoares) [inline revel/package.go]: maybe we should figure out a better, more specific name for the func?

**Comment by notzippy** [comment]
I am closing this for now, I think the better way to go with this is a rewrite see #81

**Pull request closed without merging**